iolib/drive.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+
+from .utils import build_google_api
+
+
+API_NAME = 'drive'
+API_VERSION = 'v3'
```

Comment:
Suggestion : shall we keep the constants and variable outside in another file ?

Reply:
I don't see it necessary. Also, other files contain the same constant names (see sheets pr)